src/components/MonitoringPlanEvaluationReport/MonitoringPlanEvaluationReport.js

```diff
@@ -0,0 +1,163 @@
+import React, { useEffect, useRef } from "react";
+import { Button } from "@trussworks/react-uswds";
+import { DataTableRender } from "../DataTableRender/DataTableRender";
+
+
+export const MonitoringPlanEvaluationReport = ({
+}) => {
```

Comment:
the {} should be taken out of the arguments if not being used (it'll flag it as a bug)

Reply:
Fixed, with latest push